src/pip/_internal/distributions/__init__.py

```diff
@@ -0,0 +1,25 @@
+from pip._internal.distributions.source import SourceDistribution as _SrcDist
+from pip._internal.distributions.wheel import WheelDistribution as _WhlDist
```

Comment:
Why the aliases ?

Reply:
`_names` were too long for linter to be happy and I don't want people to do from `from pip._internal.distributions import XYZ` -- mentioning Source/Wheel Distribution is okay for now.

Eventually, this file should be empty anyway. :)